docs/src/components/DocsBehaviorRoot.tsx

```diff
@@ -54,7 +54,6 @@ class DocsBehaviorRoot extends React.Component<any, any> {
                   <div style={{ display: 'flex' }}>
                     <div style={{ flex: '1', marginBottom: '1em' }}>
                       <ComponentExampleTitle
-                        id={_.kebabCase(variation.name)}
```

Comment:
This stuff is not working because `ComponentExampleTitle` doesn't spread rest props and accepts only `title` and `description`.

Reply:
this will prevent us from referencing specific example - previously it was possible to share links like the following one: https://stardust-ui.github.io/react/components/popup#types-popup-content-wrapper-example

Now I might expect that it won't work. Seems that this is not a right approach to fix this types-related issue - am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/14183168/48270065-c130ad00-e441-11e8-89c8-76ca884c1ee3.png)

This change is related to behaviours and it's is not working there.

Reply:
I will check this thing and will get it working.

Reply:
I restored the `id` on `Grid.Row`, it's the same thing as in `ComponentExample`